.pre-commit-config.yaml

```diff
@@ -1,4 +1,15 @@
 repos:
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: v8.27.0
```

Comment:
Same discussion as in #21: Do we want the same version here as in the pckage-lock.json?
At least the eslint mirror seems to be a bit more up to date, so version `8.38.0` is the latest version in both repositories.

Reply:
Ah, the `^8.27.0` in the package.json means "compatible with `8.27.0`", thus it installed `8.38.0` in the lock file (it's the same for prettier - `^2.7.1` still installs the newest available version [`2.8.7`](https://github.com/digitalfabrik/integreat-compass/blob/f016d20595bd9b365bf28fbbd3440f99c292db65/package-lock.json#L12)).
But ok, since they are compatible, a few minor versions difference shouldn't cause a different output...

Reply:
Then I guess it can stay as it is :see_no_evil: